NEWS.md

```diff
@@ -28,7 +28,10 @@
   A new field called `frontmatter_format` ("YAML", "TOML", or "JSON")
   contains the format of the frontmatter.
   (issue: #126, @maelle)
-  
+  * New `protect_fences()` function will add a `fence='true'` attribute to fences of Pandoc fenced divs to allow parsing of the XML for sending to
+  external APIs. 
+* New `$protect_fences()` method implements `protect_fences()` on yarn objects
```

Comment:
should it be called `protect_fence()`? I find the plural more natural but the other protection functions use the singular.

Reply:
I think the plural makes sense partially because `protect_curlies()` sounds slightly disgusting to my USAian ears 🤮